docs/getting-started-guides/aws.md

```diff
@@ -2,12 +2,31 @@
 assignees:
 - justinsb
 - lavalamp
+- clove
 
 ---
 
 * TOC
 {:toc}
 
+
+## Supported Production Grade Tools with High Availability Options
+
+* [Kubernetes Operations](https://github.com/kubernetes/kops) - Production Grade K8s Installation, Upgrades, and Management. Supports running Debian and CentOS in AWS.
+
+* CoreOS maintains [a CLI tool](https://coreos.com/kubernetes/docs/latest/kubernetes-on-aws.html), `kube-aws` that will create and manage a Kubernetes cluster based on [CoreOS](http://www.coreos.com), using AWS tools: EC2, CloudFormation and Autoscaling.
+
+## Other Options
+
+* Other community projects exist that use tools such that use other configuration management tooling.
+
+TODO: add more options
+
+## Legacy Tooling
+
+`kube-up.sh` is a legacy tool that is an easy way to spin up a cluster.  This tool
+is being deprecated, and does not create a production ready environment.
+
 ## Prerequisites
```

Comment:
The headers don't line up and it messes with the document's structure.

Instead of adding a separate "Legacy Tooling" header to a document that isn't set up to support, it, just add a note underneath the kube-up overview that mentions that it is a legacy tool and is being deprecated, and kube-aws is the new preferred method. You can use the blockquote to set it off:

> **Note**: `kube-up.sh` is a legacy tool and is being deprecated in the near future. `kube-aws` is the preferred method for spinning up a Kubernetes cluster on AWS.